api/terms/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" prefix="skos: http://www.w3.org/2004/02/skos/core#">
+  <head>
+    <meta charset="UTF-8">
+    <title>iRail terminology</title>
+  </head>
+  <body typeof="skos:ConceptScheme" resource="https://api.irail.be/terms/">
```

Comment:
For URIs in iRail I always pick the http URIs instead of the https ones. Can you change this?